backend/app/modules/scheduler/SchedulerActor.scala

```diff
@@ -48,18 +50,29 @@ class SchedulerActor(processes: Queries, configuration: play.api.Configuration,
     * * Download image
```

Comment:
Double stars on the comments (I think you should use full english sentence to explain what does the pipeline); id parameter is not documented